### ISSUE TITLE
Warning: This is apparently banneable

### ISSUE THREAD

**Issue by GovFox** [Soneliem/WAIUA#13]
Had been playing valorant since Beta, started using this program maybe a month ago. 

Today when logging in, I got a message on the lines of "Account is temporarily suspended for security, contact support".
Upon contacting support and asking what was going on, all I got back was:

"This account has been permanently banned due to use of prohibited third-party software and will be unable to use VALORANT. Please let me know if you have any constructive feedback. I’m sorry if this decision disappoints you, but it is final."

Before this, about 3 days ago I had someone all worried and asking how I saw their rank if they were using anonymous mode, I am guessing they reported and this is where the issues started. This is the only 3rd party program related to VALORANT I ever used, you can chose to believe that or don't but why would I risk a beta account?

**Comment by Soneliem** [comment]
After looking up similar instances of your specific ban, I have noticed that it is unfortunately not uncommon. Most I believe we're not hacking but instead were running flagged software which were not related to Valorant or Cheating in any way. However you cannot check this yourself as Riot will not provide any further information other than the one you got. That is really a frustrating practice.  

As for my app, I do not believe that it was my app, all these software bans are apps that interface directly with the game engine or files which my app does not. The API method of communicating with the game has been used by many similar apps, some with wayy more users than waiua while integrating with the API more than it. There have been no reports of bans in any of these apps. 

WAIUA does not break any sort of cheating rules and I have made sure that I follow the competitive integrity guidelines. The use of the API has also been confirmed by a Riot employee to not be bannable.

Additionally, if my app was to be banned, I would be contacted directly to stop it. It would be stupid to ban a few thousand people at the same time for an app that isn't even bannable. If I do get contacted I will comply fully as I respect their decision. 

TLDR: It is almost certain that my app was not the cause of this. I sympathise with your situation and hope that additional contact may get you unbanned.
